demo/Snakefile

```diff
@@ -8,13 +8,14 @@ import oncopipe as op
 
 SAMPLES = op.load_samples("data/samples.tsv")
 
-
 ##### REFERENCE_FILES WORKFLOW #####
 
 
 subworkflow reference_files:
     workdir:
-        "reference/"
+        # "/projects/bgrande/reference_files/"
```

Comment:
The demo Snakefile shouldn't have any lines commented out and should have all paths restored to how they are set in the Master branch. Remember your changes eventually get merged to become the new demo.

Reply:
Uncommented all